pkg/protobuf/bbgo.pb.go

```diff
@@ -0,0 +1,2122 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
maybe put the pb files in the pkg/pb package, because the protobuf name might conflict with the official import.

Reply:
done